os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -161,6 +161,7 @@ static int binary_manager_load(int bin_idx)
 	int ret;
 	int bin_count;
 	load_attr_t load_attr;
+	bool need_update_bootparam;
```

Comment:
For boolean type, please include `stdbool.h`.

Reply:
In this file, most of the name of bootparam is bp.
How about changing this variable also `need_update_bp`?